projects/pt1/e2e_testing/xfail_sets.py

```diff
@@ -1917,14 +1917,6 @@
     "TypeConversionI64ToI32Module_basic",
 
     # Failure - onnx_lowering: onnx.Clip
-    "ElementwiseClampMaxModule_basic",
-    "ElementwiseClampMinModule_basic",
-    "ElementwiseClampMinTensorFloatModule_basic",
-    "ElementwiseClampMinTensorIntModule_basic",
-    "ElementwiseClampModule_basic",
-    "ElementwiseClampTensorFloatModule_basic",
-    "ElementwiseClampTensorInt8Module_basic",
-    "ElementwiseClampTensorIntModule_basic",
     "NormalizeModule_basic",
```

Comment:
I will try to reprocess the file this week so we know what the new blocker is on this issue.